src/fdb_tsdb.c

```diff
@@ -72,6 +72,8 @@
         if (result != FDB_NO_ERR) return result;                               \
     } while(0);
 
+#pragma pack(push,1)
```

Comment:
I think it's better to not pack those structs as this will impact already stored data (will affect users that already use FlashDB)

Reply:
Yes, this will have compatibility issues.